client/src/route/auth/WaitAndNavigate.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import * as React from 'react';
+import { wait } from '../../util/auth/Authentication';
+
+const WaitNavigateAndReload = () => {
+  const [shouldNavigate, setShouldNavigate] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const waitForFiveSecondsAndNavigate = async () => {
+      await wait(5000);
```

Comment:
is this to avoid the error of unpredictable redirect? If so, can we always use a constant wait time?

Reply:
This was made in case of an auth.error which we were getting quite a lot before we fixed OIDC config. Since then we have not gotten the auth.error, however we felt it would do no harm to keep it as all it does it redirect from the error screen back to the Signin page after 5 secs.

Reply:
if you can leave a comment there, that would be helpful.